api/fs/root.go

```diff
@@ -0,0 +1,39 @@
+package apifs
```

Comment:
Why does this need to be a plugin?

Reply:
Just added a comment addressing this and other work that still needs to be done.

Reply:
Ah, for testing FUSE stuff. Should probably be a developer thing then, i.e. users should not be able to enable it.